CONTRIBUTING.md

```diff
@@ -90,12 +90,70 @@ Follow GitHub's fork-branch-pull request pattern.
 7. Provide your full name or GitHub username and your company or organization if applicable.
 
 #### Once a Pull Request Ready for Review
-1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`.
-2. Verify there is only one commit message. Squash or amend the commit messages if necessary.
+1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`. The commit message can use `Fix #XYZ, LONG_DESCRIPTION` if necessary. 
```

Comment:
Format is
```
Fix #XYZ, SHORT_DESCRIPTION

LONG_DESCRIPTION (optional)
```

Reply:
Updated format in sections Ready to Add Your Code?, Once a Pull Request Ready for Review, and Soft Reset.